wavelab/AirSim
# Custom car pawn

Adds the custom car pawn object.

Test instructions:
1. Switch to this branch and re-build Airsim (`anm_unreal_sim/submodules/AirSim/build.sh`)
2. Edit your AirSim settings from `  "SimMode": "Car",` to `  "SimMode": "CustomCar",`. 
3. Build the base_unreal_project
4. Run a scenario.

Note that the code could probably be cleaned up and improved more, but this is a version that works good.

Also, if the car tips over, the game will crash, since exported maplesim code errors. Adding a check for that in the future is recommended.
